index.js

```diff
@@ -307,8 +311,12 @@ module.exports = {
     //
     // Stylistic Issues
     //
+    // enforce linebreaks after opening and before closing array brackets
+    'array-bracket-newline': 'off',
```

Comment:
I had this set to this, but with this and array-element-newline set to the same created a strange use issue that I didn't have a good way to resolve. Then I noticed that object-curly-newline was off as well, so I just set these both to off.